packages/app/utils/constants.js

```diff
@@ -57,6 +57,7 @@ const Messages = {
   INVALID_TOKEN: "Invalid token.",
   EXPIRED_TOKEN: "Token expired.",
   VERIFICATION_FAIL: "Verification failed.",
+  EMAIL_SENT: "Verification Email sented successfully.",
```

Comment:
```suggestion
  EMAIL_SENT: "Verification Email sent successfully.",
```

Please use a spell check extension to avoid typos

Reply:
Now changed successfully:  EMAIL_SENT: "Verification Email sent successfully.",
<img width="867" height="170" alt="image" src="https://github.com/user-attachments/assets/abd894b6-63b8-4ea4-bf7a-c369410bc1f2" />
Done